tests/integration-tests/tests/networking/test_placement_group.py

```diff
@@ -0,0 +1,162 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import logging
+
+import boto3
+import pytest
+import utils
+from assertpy import assert_that
+from cfn_stacks_factory import CfnStack
+from remote_command_executor import RemoteCommandExecutor
+from troposphere import Template
+from troposphere.ec2 import PlacementGroup
+from utils import generate_stack_name
+
+from tests.common.scaling_common import _get_asg
+
+
+@pytest.mark.regions(["af-south-1"])
+@pytest.mark.instances(["c5.xlarge"])
+@pytest.mark.schedulers(["slurm", "sge"])
+@pytest.mark.oss(["alinux2"])
+@pytest.mark.usefixtures("os", "instance", "scheduler", "region")
+def test_existing_placement_group_in_cluster(
+    region, scheduler, pcluster_config_reader, clusters_factory, placement_group_stack, instance
+):
+    """Test the case when placement_group is provided. This test is not for awsbatch scheduler."""
+    placement_group = placement_group_stack.cfn_resources["PlacementGroup"]
+    cluster_config = pcluster_config_reader(placement_group=placement_group)
+    cluster = clusters_factory(cluster_config)
+    _assert_placement_group(cluster, scheduler, region, placement_group, None, instance)
```

Comment:
I missed this previously, looks like this test uses `placement = compute`, can we do an assertion to make sure the Head node is not in a placement group?

Reply:
Yes, we can. Define a new function `_check_head_node_for_compute_type(remote_command_executor, region)` to check head instance not in a placement group when placement type is compute

Reply:
Great, now that looks a lot like the code we are using to check that head node is in the placement group in `test_dynamic_placement_group_in_cluster` can we just put them all in the same function and have a flag so we can know what to assert?

The function can look something like this:
```
def _check_head_node_placement_group(remote_command_executor, region, expected_placement_group=None):
    ...
    #somethingsomethingsomething
    ...
    if expected_placement_group:
        assert_that(head_node_placement_group).is_equal_to(expected_placement_group)
    else:
        assert_that(head_node_placement_group).is_equal_to("")
```

Reply:
Good idea! Done